test/unit/adapters/http.js

```diff
@@ -261,9 +259,7 @@ describe('supports http with nodejs', function () {
       axios.head('http://localhost:4444/one').then(function (res) {
         assert.equal(res.status, 200);
         done();
-      }).catch(function (err) {
-        done(err);
-      });
+      }).catch(done);
```

Comment:
This implementation was correct but I changed it to the compact form for consistency with all other tests.